components/sst_importer/src/sst_importer.rs

```diff
@@ -981,6 +1014,41 @@ mod tests {
         })
     }
 
+    fn create_sample_external_kv_file() -> Result<(tempfile::TempDir, StorageBackend, KvMeta)> {
+        let ext_dir = tempfile::tempdir()?;
+        let file_name = "v1/t000001/abc.log";
+        let file_path = ext_dir.path().join(file_name);
+        std::fs::create_dir_all(file_path.parent().unwrap())?;
```

Comment:
should remove temporary file after test finished?

Reply:
The temp dir will be delete automatically when the `temp dir` variable is recycled.
https://github.com/Stebalien/tempfile/blob/66aa57f7d3a38234fcd393077366b61d36171e42/src/dir.rs#L404